src/mscorlib/corefx/Interop/Unix/System.Native/Interop.GetRandomBytes.cs

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Same for ArrayPoolEventSource.cs. There are just a few files left under corefx. You can just move all of them so that the directory disappears.